tensorflow/tools/ci_build/builds/pip.sh

```diff
@@ -125,27 +136,46 @@ echo "whl file path = ${WHL_PATH}"
 # Install, in user's local home folder
 echo "Installing pip whl file: ${WHL_PATH}"
 
-# Create temporary directory for install test
+# Create virtualenv directory for install test
 VENV_DIR="${PIP_TEST_ROOT}/venv"
-rm -rf "${VENV_DIR}" && mkdir -p "${VENV_DIR}"
-echo "Create directory for virtualenv: ${VENV_DIR}"
+if [[ -d "${VENV_DIR}" ]] &&
+   [[ ! -z "${TF_BUILD_NO_CACHING_VIRTUALENV}" ]] &&
+   [[ "${TF_BUILD_NO_CACHING_VIRTUALENV}" != "0" ]]; then
+  echo "TF_BUILD_NO_CACHING_VIRTUALENV=${TF_BUILD_NO_CACHING_VIRTUALENV}:"
+  echo "Removing existing virtualenv directory: ${VENV_DIR}"
+
+  rm -rf "${VENV_DIR}" || \
+      die "Failed to remove existing virtualenv directory: ${VENV_DIR}"
+fi
```

Comment:
Ok, this style is here on multiple places. But for next time... why not simple remove all these features. The scripts can be considerably simpler by removing all this. And one still have it all. Look...

```bash
export TF_BUILD_NO_CACHING_VIRTUALENV=1
tensorflow/tools/ci_build/builds/pip.sh CPU
```

vs

```bash
rm -rf pip_test/venv
tensorflow/tools/ci_build/builds/pip.sh CPU
```

It is shorter even outside of he script and you have one less api feature (TF_BUILD_NO_CACHING_VIRTUALENV) to support. 10+ lines of script less.